tests/pipelines/helper_functions.py

```diff
@@ -80,6 +83,17 @@ def train_log_and_register_model(model_name, is_dummy=False):
     return "models:/{model_name}/Production".format(model_name=model_name)
 
 
+def get_execution_directory_location(pipeline_root_path, custom_execution_directory=None):
+    if custom_execution_directory:
+        return pathlib.Path(custom_execution_directory)
+    return (
+        pathlib.Path.home()
+        / ".mlflow"
+        / "pipelines"
+        / _get_execution_directory_basename(pipeline_root_path)
```

Comment:
Can we use `get_or_create_base_execution_directory` instead? 
https://src.dev.databricks.com/mlflow/mlflow/-/blob/mlflow/pipelines/utils/execution.py?L224&subtree=true